src/NzbDrone.Core/Download/Pending/PendingReleaseService.cs

```diff
@@ -67,7 +66,7 @@ public class PendingReleaseService : IPendingReleaseService,
         }
 
 
-        public void Add(DownloadDecision decision)
+        public void Add(DownloadDecision decision, PendingReleaseReason reason)
         {
             var alreadyPending = GetPendingReleases();
```

Comment:
What if the release was first Delayed, but then hit a DownloadClientUnavailable?
I think it should update the ECT and Reason. (Can't use Upsert, since that would replace the Added date.)

I'd also give the Debug log entries a bit more info on the type of Pend.